e2e-tests/tests/tls-cert-manager/00-deploy-cert-manager.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+timeout: 120
+commands:
+  - script: |-
+      set -o xtrace
+
+      source ../../functions
+
+      deploy_cert_manager
```

Comment:
we should put an assertion to this step, at least to check if cert-manager exists and it's deployments are ready

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/125/commits/19b28f3153b44efb388d119b814622608cb4b84d